packages/fastboot/src/sandbox.js

```diff
@@ -17,9 +17,11 @@ module.exports = class Sandbox {
     let URL = require('url');
     let globals = this.globals;
 
+    const sourceMapConfig = process.setSourceMapsEnabled ? {} : { sourceMapSupport };
```

Comment:
This has a few issues:

* when we are using `process.setSourceMapsEnabled` we should not expose any globals
* we should not be changing the name of the global to export (when using Node < 14) see (which uses the old global for exposing `source-map-support`):

https://github.com/ember-fastboot/ember-cli-fastboot/blob/f94426759fb89d94db71b93f97cae92fb183e114/packages/fastboot/src/scripts/install-source-map-support.js#L2-L6


My suggestion is (basically):

```js
    const sourceMapConfig = process.setSourceMapsEnabled ? null : { sourceMapSupport };

    let sandbox = Object.assign(
      sourceMapConfig,
      {
        console,
        setTimeout,
        clearTimeout,
        URL,
        // Convince jQuery not to assume it's in a browser
        module: { exports: {} },
      },
      globals
    );
```

This will:

1. do nothing if we are going to use `process.setSourceMapsEnabled`
2. when we are using the older system, we still expose the `source-map-support` package at the correct location